demo/src/main/java/com/jira/demo/repository/SquadRepo.java

```diff
@@ -2,5 +2,6 @@
 
 import com.jira.demo.model.Squad;
 import org.springframework.data.jpa.repository.JpaRepository;
+
```

Comment:
Don't forget to format your code with combination of `Ctrl + Alt + L` to keep your code clean and consistent